src/main/java/Task.java

```diff
@@ -0,0 +1,43 @@
+public class Task {
+  public String string;
+  public Boolean mark;
+  public Boolean todo;
+  public Boolean deadline;
+  public Boolean event;
+  public String time;
+  public String startTime;
+  public String endTime;
```

Comment:
The name of the fields can be improved to follow the Name well guideline. Such as string can be replace with str and mark can be replace with isMark.